spec/services/shops_list_service_spec.rb

```diff
@@ -24,10 +27,16 @@
       expect(shop.association(:logo_attachment).loaded?).to be true
       expect(shop.logo.association(:blob).loaded?).to be true
     end
+
+    it "only fetches enterprises with an active order cycle" do
+      open_enterprise_ids = Enterprise.distributors_with_active_order_cycles.pluck(:id).to_set
+      expect(open_enterprise_ids).not_to be_empty
+      expect(shops.pluck(:id)).to all be_in open_enterprise_ids
```

Comment:
It's good to cover the code but the expectation doesn't achieve much. If you call `Enterprise.distributors_with_active_order_cycles` in the spec and in the code under test and check if they equal then the test will always pass. A better test would be to manually select the enterprise ids from your test data.

So I guess that `distributors[0]` was chosen to have an order cycle here. Reading the spec, I would expect that id to be returned. Can we check that in the spec?